test/runnable/imports/test18322import.d

```diff
@@ -0,0 +1,20 @@
+module test18322import;
+void fun(string templateFileFullPath = __FILE_FULL_PATH__,
+    string templateFile = __FILE__)(string fileFullPath = __FILE_FULL_PATH__)
+{
+    version(Windows)
+    {
+        assert(fileFullPath[1..3] == ":\\");
+        assert(templateFileFullPath[1..3] == ":\\");
+        enum lastPart = "test\\runnable\\test18322.d";
+    }
+    else
+    {
+        assert(fileFullPath[0] == '/');
+        assert(templateFileFullPath[0] == '/');
+        enum lastPart = "test/runnable/test18322.d";
+    }
```

Comment:
Hmm, how about just checking for the end of the file name?
- I'm not sure whether LDC or GDC use a similar setup to run the test suite
- It would allow convenient execution of the test with `rdmd` or `dmd -i`

Reply:
Also your test at https://github.com/dlang/dmd/blob/master/test/compilable/line.d doesn't uses the `test` directory.

Reply:
ok, I removed the `test` part of the path.  I think having `runnable/<test-name>` is probably ok.